system/jlib/jfile.cpp

```diff
@@ -779,7 +779,7 @@ void CFile::move(const char *newname)
 #ifdef _WIN32
     unsigned retry = 0;
     for (;;) {
-        if (MoveFileEx(filename.get(),newname,0)) 
+        if (MoveFileEx(filename.get(),newname, MOVEFILE_REPLACE_EXISTING))
```

Comment:
I think this functionality is more correct - we assume that move will replace any existing file, since that is the linux functionality.